### PR TITLE
Successfully separated the front end from the back end

### PR DESCRIPTION
**Pull request by AndCrit**
Added a server file and added some dependencies

### CODE REVIEW

**Review by IqbalBan — APPROVED** [review]
passable i guess

**Comment by SachJay** [comment]
Ankit once we get 2 reviewers, when you press the merge button make sure its set it `Squash and merge`.
image

**Review by SachJay — APPROVED** [review]
ima just approve its been a while, ur good to merge

**Pull request merged**
